core_backend/app/question_answer/models.py

```diff
@@ -200,19 +206,67 @@ def __repr__(self) -> str:
         )
 
 
-async def save_feedback_to_db(
-    feedback: FeedbackBase,
+async def save_response_feedback_to_db(
+    feedback: ResponseFeedbackBase,
+    asession: AsyncSession,
+) -> ResponseFeedbackDB:
+    """
+    Saves feedback to the database.
+    """
+    response_feedback_db = ResponseFeedbackDB(
+        feedback_datetime_utc=datetime.utcnow(),
+        feedback_sentiment=feedback.feedback_sentiment,
+        query_id=feedback.query_id,
+        feedback_text=feedback.feedback_text,
+    )
+    asession.add(response_feedback_db)
+    await asession.commit()
+    await asession.refresh(response_feedback_db)
+    return response_feedback_db
+
+
+class ContentFeedbackDB(Base):
+
+    __tablename__ = "content_feedback"
+
+    feedback_id: Mapped[int] = mapped_column(
+        Integer, primary_key=True, index=True, nullable=False
+    )
+    feedback_sentiment: Mapped[str] = mapped_column(String, nullable=True)
+    query_id: Mapped[int] = mapped_column(Integer, ForeignKey("user-queries.query_id"))
+    feedback_text: Mapped[str] = mapped_column(String, nullable=True)
+    feedback_datetime_utc: Mapped[datetime] = mapped_column(DateTime, nullable=False)
+    content_id: Mapped[int] = mapped_column(Integer, ForeignKey("content.content_id"))
+
+    query: Mapped[UserQueryDB] = relationship(
+        "UserQueryDB", back_populates="content_feedback", lazy=True
+    )
+
+    content: Mapped["ContentDB"] = relationship("ContentDB")
```

Comment:
Note that this does not back populate. To retrieve all the feedback comments for a content, you need to do a join. I think this is fine for now. It saves us from circular import hell. And retrieving huge lists of feedback through content.

If we find a use case where we need this, we can add it ... and refactor.